chatkit-core/src/main/kotlin/com/pusher/chatkit/cursors/CursorService.kt

```diff
@@ -50,44 +48,37 @@ internal class CursorService(private val chatManager: ChatManager) {
     }
 
     fun getReadCursor(userId: String, roomId: Int) : Future<Result<Cursor, Error>> =
-        (cursors[userId][roomId]?.asSuccess<Cursor, Error>() ?: notSubscribedToRoom("$roomId").asFailure())
+        (cursorsStore[userId][roomId]?.asSuccess<Cursor, Error>() ?: notSubscribedToRoom("$roomId").asFailure())
             .toFuture()
 
     private fun notSubscribedToRoom(name: String) =
-        Errors.other("Must be subscribed to room $name to access member's read cursors")
+        Errors.other("Must be subscribed to room $name to access member's read cursorsStore")
```

Comment:
failure of Android Studio's ~context aware refactor~ naive search and replace tool